settings.gradle

```diff
@@ -9,8 +13,11 @@ plugins {
     id 'io.micronaut.build.shared.settings' version '7.0.1'
 }
 
-rootProject.name = 'project-template-parent'
+// XXX GP - groupd and artefact id are somehow derived, but how exactly?
```

Comment:
see the root gradle.properties file which defines the group id. The artefact id is the same as each module name in the include definitions below